<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageIden</class>
 <widget class="QMainWindow" name="ImageIden">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>450</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <italic>false</italic>
    <bold>true</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>图像识别</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonQuit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>360</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPicture">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>250</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>471</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>舵机控制</string>
    </property>
    <widget class="QPushButton" name="btUp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>100</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Micro Hei</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btLeft">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>100</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Micro Hei</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btRight">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>100</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Micro Hei</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btDown">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>100</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Micro Hei</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leHorizontal">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leVertical">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>110</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>垂直角度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>水平角度</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>图片存储</string>
    </property>
    <widget class="QLineEdit" name="leName">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leNumber">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>名字</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>数量</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>延时</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leTime">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>70</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>(s)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btSave">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>100</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>WenQuanYi Micro Hei</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="rbRefrashImg">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>定时刷新图像</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionFromFile"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionIden">
   <property name="text">
    <string>识别</string>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
    </font>
   </property>
  </action>
  <action name="actionZuoqu">
   <property name="text">
    <string>zhuqu</string>
   </property>
  </action>
  <action name="actionXunliang">
   <property name="text">
    <string>xunliang</string>
   </property>
  </action>
  <action name="actionFromFile">
   <property name="text">
    <string>载入图片</string>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
    </font>
   </property>
  </action>
  <action name="actionCatch">
   <property name="text">
    <string>捕获</string>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
    </font>
   </property>
  </action>
  <action name="actionTrain">
   <property name="text">
    <string>训练</string>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
    </font>
   </property>
  </action>
  <action name="actionFace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>人脸识别</string>
   </property>
   <property name="iconText">
    <string>人脸识别</string>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
    </font>
   </property>
  </action>
  <action name="actionDigit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>数字识别</string>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
    </font>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei</family>
    </font>
   </property>
  </action>
  <action name="actionAa">
   <property name="text">
    <string>aa</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
